WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -299,18 +317,19 @@ public void onClick(View v) {
     }
 
     private void initSmartLockForPasswords() {
-        if (!isGooglePlayServicesAvailable()) {
+        if (!isSmartLockAvailable()) {
```

Comment:
Nope we should keep the `isGooglePlayServicesAvailable` test here , `mCredentialsClient` will always be null